src/crypto.js

```diff
@@ -27,6 +27,20 @@ function _validateOpts({ encryptionKey }) {
   }
 }
 
+function _validateAADType(aad) {
```

Comment:
```suggestion
function _validateAAD(aad) {
```
We're validating the value, type and length of the string.